### PR TITLE
GitLab Pipeline fixes to get the tests runing

### PR DESCRIPTION
**Pull request by lluiscampos**
This should fix the CI build errors

Changelog: None

Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ variables:
   REPO_NAME: github.com/mendersoftware/mender-demo-artifact
   GITHUB_RELEASE_BINARY: mender-demo-artifact
   GITHUB_RELEASE_DEPLOY_REPO: mendersoftware/mender-demo-artifact
-  MENDER_VERSION: '2.0.0'
+  MENDER_VERSION: 'master'
```
Comment (kacf) [inline .gitlab-ci.yml]: In order for the local install to match this, I think [this default](https://github.com/mendersoftware/mender-demo-artifact/blob/master/Dockerfile#L27) should be the same. That is what will be tested during the release process, what's in here will only be used for the final upload.

Comment (lluiscampos) [reply .gitlab-ci.yml]: Mmmm. I don't understand your comment.

I changed it to align GitLab and Travis configuration, as GitLab runs are failing with these kind of messages:
```
Sep 04 08:47:12 qemux86-64 systemd[1]: Started mender-demo-artifact.service.
Sep 04 08:47:25 qemux86-64 entrypoint.sh[507]: cat: /sys/class/net/eth0/address: No such file or directory
Sep 04 08:47:25 qemux86-64 entrypoint.sh[507]: /var/www/localhost/entrypoint.sh: line 27: ../busybox: No such file or directory
Sep 04 08:47:25 qemux86-64 systemd[1]: mender-demo-artifact.service: Main process exited, code=exited, status=127/n/a
```

So I suspected that required changes were made in the first 2.1.x and pointing to master could fix this.

Comment (lluiscampos) [reply .gitlab-ci.yml]: And now with this change the tests finally work!! :)

Comment (kacf) [reply .gitlab-ci.yml]: My point is that part of the release process is to build this artifact locally, and test it. But when you do that, the value is taken from [here](https://github.com/mendersoftware/mender-demo-artifact/blob/master/Dockerfile#L27) and not from the variable in `.gitlab-ci.yml`. So you will be testing using an old version, and then, when you tag, you will build a version based on master, which you have not tested.

We are not pushing from Gitlab ATM, but I assume we will eventually.

Comment (lluiscampos) [reply .gitlab-ci.yml]: Thanks. Now I understand.

Then, that is a problem that we have both here and in Jenkins.

Comment (kacf) [reply .gitlab-ci.yml]: Ok. I guess we just need to be mindful about this when we release. This looks enough for now.

**Pull request merged**